tests/test_experimental/test_prediction_intervals/test_base.py

```diff
@@ -215,17 +215,6 @@ def test_valid_params_sampling(product_level_constant_hierarchical_ts, pipeline)
     assert_sampling_is_valid(intervals_pipeline=intervals_pipeline, ts=product_level_constant_hierarchical_ts)
 
 
-@pytest.mark.parametrize(
```

Comment:
Why this test was removed ?

Reply:
Because in #432 we added implementation for `params_to_tune`, so this doesn't raise `NotImplementedError` anymore.

Reply:
I think it is better to rewrite this test with the stub, since it is the only test for ensembles `params_to_tune`